beam-core/Database/Beam/Backend/SQL/BeamExtensions.hs

```diff
@@ -181,7 +182,7 @@ class BeamSqlBackend be => BeamHasInsertOnConflict be where
 
 newtype InaccessibleQAssignment be = InaccessibleQAssignment
   { unInaccessibleQAssignment :: [(BeamSqlBackendFieldNameSyntax be, BeamSqlBackendExpressionSyntax be)]
-  } deriving (Semigroup, Monoid)
+  } deriving (Data.Semigroup.Semigroup, Monoid)
```

Comment:
Why is this fully qualified?

Reply:
To suppress unused import warning on GHC-8.8